exasol_script_languages_container_tool/cli/commands/upload.py

```diff
@@ -42,6 +42,7 @@
 @add_options(docker_repository_options)
 @add_options(system_options)
 @add_options(luigi_logging_options)
+@click.option("--ignore-certificate/--no-ignore-certificate", default=False)
```

Comment:
In PEC, WE use 

```
@click.option('--ssl-cert-path', type=str, default="")
@click.option('--use-ssl-cert-validation/--no-use-ssl-cert-validation', type=bool, default=True)
```

We should do the same here.

https://github.com/exasol/python-extension-common/blob/822762ae0359744a21a6f485bad675fbe281f9d7/exasol/python_extension_common/deployment/language_container_deployer_cli.py#L147

verify for BucketFS Path is then

```
verify = ssl_trusted_ca or use_ssl_cert_validation
```

https://github.com/exasol/python-extension-common/blob/822762ae0359744a21a6f485bad675fbe281f9d7/exasol/python_extension_common/deployment/language_container_deployer.py#L317

Reply:
Maybe we should also align the remaining parameters to the language container deployer 

https://github.com/exasol/python-extension-common/blob/822762ae0359744a21a6f485bad675fbe281f9d7/exasol/python_extension_common/deployment/language_container_deployer_cli.py#L115

potentially actually with providing old and new parameter named, but add deprecation warnings for the old ones

Reply:
Done for the first comment. The second one I will do in a separate ticket: created #230